data/category-analytics

```diff
@@ -0,0 +1,5 @@
+# List of all domains being used in log collection or monitor.
+
+analytics.map.qq.com
+app-measurement.com
+in.appcenter.ms
```

Comment:
Would you mind providing this domain's purpose? I found it should be Microsoft App Center SDK. So it should not be kindly analysis tracker?